lib/index.d.ts

```diff
@@ -82,7 +77,7 @@ export interface BinaryResult {
     data: Uint8Array;
     filename?: string;
     signatures?: {
-        keyid: type.keyid.Keyid;
+        keyid: KeyID;
         verified: Promise<boolean>;
         valid: boolean;
```

Comment:
Is `valid` still there? (removed in v5)
Probably best to reference the openpgp.js SignResult type directly